### PR TITLE
feat/prettier complete

### PR DESCRIPTION
**Pull request by new5558**
- add lint staged: automatically run prettier and eslint on commit
-  format all file with eslint and prettier
- extra: add code splitting to modal [example](https://vuejsdevelopers.com/2017/07/08/vue-js-3-ways-code-splitting-webpack/) Note: example using vue2 syntax, but you should get the idea.
- extra: add preconnect to google fonts to improve page load speed

### CODE REVIEW

File: src/commons/constant/thailand-address/district.json [hunk 1]
```diff
@@ -1 +1,7426 @@
...
\ No newline at end of file
+[
```
Comment (new5558) [inline src/commons/constant/thailand-address/district.json:1]: Do we really need this file since we already has district.ts

**Comment by new5558** [comment]
After apply code-splitting, we can see a slight increase in Lighthouse score by 10. However, there are further possible improvements. We will talk about that after the main features are finished.
image
image

If you want  to measure yourself: https://web.dev/measure/

**Review by Seqlet — APPROVED** [review]
LGTM!

File: src/commons/constant/thailand-address/district.json (hunk 1 above)
Comment (poom-kitti) [reply src/commons/constant/thailand-address/district.json:1]: Poom forget to delete

**Pull request merged**